### PR TITLE
Updated timer docs

### PR DESCRIPTION
**Pull request by TapGhoul**
Updated the documentation for the timer to add a missing function call's description

### CODE REVIEW

**Comment by cbscribe** [comment]
I know you were following what was already written, but this description (as well as the others you didn't change) does not conform to the [writing guidelines](http://docs.godotengine.org/en/latest/community/contributing/docs_writing_guidelines.html).

- Use present tense
- Use "If true, ..." construction for booleans

For example:

> Returns [code]true[/code] if the timer is stopped.

Can you change the others as well?

**Comment by Silvea12** [comment]
Alright, I'll switch that over!

File: doc/classes/Timer.xml [hunk 1]
```diff
@@ -15,20 +15,21 @@
...
-				Return the time left for timeout in seconds if the timer is active, 0 otherwise.
+				Returns the time left for timeout in seconds if the timer is active, 0 otherwise.
 			</description>
 		</method>
 		<method name="is_paused" qualifiers="const">
 			<return type="bool">
 			</return>
 			<description>
-				Return if the timer is paused or not.
+				If [code]true[/code], the timer is paused.
```
Comment (cbscribe) [inline doc/classes/Timer.xml]: Remove comma.

"If true" is for members. This is a function, so "Returns..." is still the correct construction.

Returns [code]true[/code] if the timer is paused.

File: doc/classes/Timer.xml [hunk 2]
```diff
@@ -15,20 +15,21 @@
 			<return type="float">
 			</return>
 			<description>
-				Return the time left for timeout in seconds if the timer is active, 0 otherwise.
+				Returns the time left for timeout in seconds if the timer is active, 0 otherwise.
```
Comment (cbscribe) [inline doc/classes/Timer.xml]: "time left for timeout" sounds a bit awkward:

Returns the timer's remaining time in seconds. Returns 0 if the timer is inactive.

File: doc/classes/Timer.xml [hunk 3]
```diff
@@ -15,20 +15,21 @@
...
 			<description>
-				Return if the timer is paused or not.
+				If [code]true[/code], the timer is paused.
 			</description>
 		</method>
 		<method name="is_stopped" qualifiers="const">
 			<return type="bool">
 			</return>
 			<description>
+				If [code]true[/code], the timer is stopped.
```
Comment (cbscribe) [inline doc/classes/Timer.xml]: see above

File: doc/classes/Timer.xml [hunk 4]
```diff
@@ -37,14 +38,17 @@
 			<argument index="0" name="paused" type="bool">
 			</argument>
 			<description>
-				Set whether the timer is paused or not. A paused timer will be inactive until it is unpaused again.
+				Pause the timer. If [code]paused[/code] is [code]true[/code], the timer will not process until it is started or unpaused again, even if [code]start[/code] is called.
```
Comment (cbscribe) [inline doc/classes/Timer.xml]: Pause -> Pauses

Use method links: change "[code]start[/code]" to [method start]

File: doc/classes/Timer.xml [hunk 5]
```diff
@@ -37,14 +38,17 @@
...
+				Default: [code]false[/code]
```
Comment (cbscribe) [inline doc/classes/Timer.xml]: Remove this - this method returns `void`, so it can't have a default value.

File: doc/classes/Timer.xml [hunk 6]
```diff
@@ -37,14 +38,17 @@
...
 			</description>
 		</method>
 		<method name="start">
 			<return type="void">
 			</return>
 			<description>
 				Start the Timer.
+				This also resets the timeout to [code]wait_time[/code].
```
Comment (cbscribe) [inline doc/classes/Timer.xml]: Start -> Starts

Don't capitalize "Timer"

Starts the timer and resets the timeout to [member wait_time].

File: doc/classes/Timer.xml [hunk 7]
```diff
@@ -37,14 +38,17 @@
...
+				If the timer is paused, the timer will not resume automatically.
```
Comment (cbscribe) [inline doc/classes/Timer.xml]: I'm not sure this is getting across the point, which is that `start()` doesn't resume a paused timer. Maybe:

Note: this method will not resume a paused timer. See [method set_paused].

**Review by cbscribe — CHANGES_REQUESTED** [review]

File: doc/classes/Timer.xml (hunk 4 above)
Comment (Silvea12) [reply doc/classes/Timer.xml]: Ah, this was not found within http://docs.godotengine.org/en/latest/community/contributing/docs_writing_guidelines.html - maybe this documentation should be updated too.

**Comment by Silvea12** [comment]
@cbscribe The changes have been made. I also left a note in one of your change requests, one thing you mentioned was not documented in the guidelines page.

File: doc/classes/Timer.xml (hunk 4 above)
Comment (cbscribe) [reply doc/classes/Timer.xml]: http://docs.godotengine.org/en/latest/community/contributing/updating_the_class_reference.html#improve-formatting-with-bbcode-style-tags

File: doc/classes/Timer.xml [hunk 8]
```diff
@@ -37,14 +38,16 @@
...
-				Set whether the timer is paused or not. A paused timer will be inactive until it is unpaused again.
+				Pauses the timer. If [code]paused[/code] is [code]true[/code], the timer will not process until it is started or unpaused again, even if [method start] is called.
 			</description>
 		</method>
 		<method name="start">
 			<return type="void">
 			</return>
 			<description>
-				Start the Timer.
+				Start the timer.
```
Comment (cbscribe) [inline doc/classes/Timer.xml]: Start -> Starts.

I also think you don't need the newline here. How about:

> Starts the timer and resets the remaining time to [code]wait_time[/code].

**Comment by Silvea12** [comment]
Updated, as requested.

**Review by cbscribe — APPROVED** [review]

**Comment by cbscribe** [comment]
Thanks for your contribution!

**Pull request merged**